src/ApiService/ApiService/Functions/ReproVmss.cs

```diff
@@ -15,7 +15,7 @@ public class ReproVmss {
         _context = context;
     }
 
-    [Function("repro_vms")]
+    [Function("ReproVmss")]
     public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE")] HttpRequestData req) {
```

Comment:
And a route here as well

Reply:
Wait why? This isn't necessary for the other functions.